packages/manager/modules/server-sidebar/src/order.constants.js

```diff
@@ -262,36 +262,22 @@ export const ORDER_URLS = {
         'https://www.ovhcloud.com/en/network-security/ovhcloud-connect/provider/',
     },
     dbaas_logs: {
-      FR:
-        "https://www.ovh.com/fr/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      CZ:
-        "https://www.ovh.cz/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      DE:
-        "https://www.ovh.de/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
-      ES:
-        "https://www.ovh.es/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
+      FR: 'https://www.ovh.com/manager/cloud/repsac/#/dbaas/logs/welcome',
+      CZ: 'https://www.ovh.cz/manager/cloud/repsac/#/dbaas/logs/welcome',
```

Comment:
The  `https://www.ovh.cz/manager/cloud/repsac` doesn't exist
I think here we need to consider the URL depending on the region (EU,CA and US) rather on the subsidiary``
Also this module has been moved to dedicated in the branch you target so it will be 
`https://www.ovh.com/manager/dedicated/#/dbaas/logs/welcome`

Reply:
thanks ! i fix it